src/modules/search/components/SearchResultCard.tsx

```
@@ -34,6 +38,12 @@ export default function SearchResultCard({ note }: { note: NoteSearch }) {
             }).format(note.updatedAt)}
           </div>
         </CardHeader>
+        <CardContent>
+          <p
+            className="text-sm border-l-4 py-2 pl-2"
+            dangerouslySetInnerHTML={{ __html: sanitizedSnippet }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)